app/javascript/src/scenes/SettingsPage/scenes/profile/components/ProfileInputField.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import { string, func } from "prop-types";
+
+import InputDisabled from "../../../components/InputDisabled";
```

Comment:
why can't you create one Input component for enabled/disabled state? point of React is DRY and this creates a lot of it.